docroot/modules/contrib/ova/ova.inc

```diff
@@ -0,0 +1,26 @@
+<?php
+
+/**
+ * @file
+ * Open Video Annotation using template variables.
+ */
+
+ /**
+ *  Implements template_preprocess_hook().
+ */
+function template_preprocess_ovaAudio(&$variables) {
+  $annotation_settings = array();
```

Comment:
Javascript using annotation_settings variable is commented out under helper.js. Not sure if this is still needed. Cleanup needed?